src/linter/include/sourcemeta/blaze/linter.h

```diff
@@ -45,6 +48,7 @@ class SOURCEMETA_BLAZE_LINTER_EXPORT ValidExamples final
 #pragma warning(disable : 4251)
 #endif
   const Compiler compiler_;
+  mutable std::set<std::size_t> invalid_indices_;
```

Comment:
Can we make this just a `std::vector`? I don't think we benefit much from the semantics of a set (as we guarantee that we get unique indexes), and thus we can avoid the equality checks from `std::set`